src/main/resources/application.properties

```diff
@@ -29,6 +29,9 @@ application.ecc.issuer.connector=http://w3id.org/engrd/connector/
 #Use default behavior for contract agreement, should not be used in production
 application.contract.negotiation.demo=true
 
+#mydata or platoon
+application.usageControlVersion=platoon
+
```

Comment:
Besides this, I saw that we are also missing  
application.extractPayloadFromResponse=

Reply:
Good one
Added in runtime and test scope